test/misc-tests.jl

```diff
@@ -7,11 +7,15 @@
 end
 
 # Test rad2sec
-@test rad2sec.([1]) ≈ [206264.80624709636]
-@test rad2sec(pi) ≈ 648000.0
-@test sec2rad(rad2sec(12.34)) ≈ 12.34
+@testset "rad2sec" begin
+    @test rad2sec.([1]) ≈ [206264.80624709636]
```

Comment:
Trivial dot calls like this, with a one-element vector, can be simplified to `radsec(1)`.  The test with vector input made sense before the dot syntax was introduced.  Dot calls are still useful when we want to test a bunch of different arguments at once.

I should have made this change as part of this commit https://github.com/JuliaAstro/AstroLib.jl/commit/9b0430116af14252f3bdf88b0caa7f49630ea361